nexus/lib/pwscf_analyzer.py

```diff
@@ -140,35 +140,37 @@ def analyze(self):
 
         outfile = os.path.join(path,outfile_name)
 
-        # perform MD analysis
-        f = TextFile(outfile)
-        n = 0
-        md_res = []
-        while f.seek('!',1)!=-1:
-            E = float(f.readtokens()[-2])
-            f.seek('P=',1)
-            P = float(f.readtokens()[-1])
-            f.seek('time      =',1)
-            t = float(f.readtokens()[-2])
-            f.seek('kinetic energy',1)
-            K = float(f.readtokens()[-2])
-            f.seek('temperature',1)
-            T = float(f.readtokens()[-2])
-            md_res.append((E,P,t,K,T))
-            n+=1
-        #end while
-        md_res = array(md_res,dtype=float).T
-        quantities = ('total_energy','pressure','time','kinetic_energy',
-                      'temperature')
-        md = obj()
-        for i,q in enumerate(quantities):
-            md[q] = md_res[i]
-        #end for
-        md.potential_energy = md.total_energy - md.kinetic_energy
-        self.md_data = md
-        self.md_stats = self.md_statistics()
-        if self.info.md_only:
-            return
+        if self.input.control.calculation != 'nscf':
```

Comment:
Sorry, this was not meant to be left exposed.  Please place within a try-except block similar to the others below instead.